CMakeLists.txt

```diff
@@ -389,6 +393,83 @@ endif()
 
 # ##############################################################################
 
+# If cuDSS is enabled and environment variables are set, install the cuDSS version
+if(USE_CUDSS AND DEFINED ENV{CUDA_PATH} AND DEFINED ENV{CUDSS_PATH})
+
+  # Force 32-bit integers for cuDSS compatibility
+  if(DLONG)
+    message(FATAL_ERROR "cuDSS requires 32-bit integers. Set DLONG=OFF or do not use -DDLONG=ON")
+  endif()
+
+  message(STATUS "CUDA_PATH set to $ENV{CUDA_PATH}")
+  message(STATUS "CUDSS_PATH set to $ENV{CUDSS_PATH}")
+  message(STATUS "Will install SCS-cuDSS (libscscudss).")
+
+  # Enable CUDA language
+  enable_language(CUDA)
```

Comment:
As we are not compiling `.cu` code directly in the project, I think this can be removed.

```suggestion
```